src/main/java/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -99,18 +99,23 @@ private static void populateAvailableGames(final Map<String, URI> availableGames
     return Arrays.asList(files);
   }
 
-  private static void populateFromDirectory(final File mapDir, final Map<String, URI> availableGames,
-      final Set<String> availableMapFolderOrZipNames, final Set<String> mapNamePropertyList) {
+  private static void populateFromDirectory(
+      final File mapDir,
+      final Map<String, URI> availableGames,
+      final Set<String> availableMapFolderOrZipNames,
+      final Set<String> mapNamePropertyList) {
     final File games = new File(mapDir, "games");
     if (!games.exists()) {
       // no games in this map dir
       return;
     }
-    for (final File game : games.listFiles()) {
-      if (game.toURI() != null && game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
-        final boolean added = addToAvailableGames(game.toURI(), availableGames, mapNamePropertyList);
-        if (added) {
-          availableMapFolderOrZipNames.add(mapDir.getName());
+    if (games.listFiles() != null) {
+      for (final File game : games.listFiles()) {
+        if (game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
+          final boolean added = addToAvailableGames(game.toURI(), availableGames, mapNamePropertyList);
+          if (added) {
```

Comment:
I think it is maybe debatable either way.  I'm partly inclined to update it, but at the same time this is just an indentation update here: https://github.com/triplea-game/triplea/pull/1972/files?w=1 

`addToAvailableGames` I think is a problematic method though, I think part of the reason it was on two lines was to convey that `addToAvailableGames` has a lot of side effects.

IMO not worth messing around with these two lines just yet unless we would do some deeper stuff.